src/components/Login.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Form, Input } from 'antd';
+import React, { useState, useContext } from 'react';
+import { createTextChangeRange } from 'typescript';
+import { ContextInterface, LoginContext } from '../contexts/LoginContext';
+
+const changeIsLoggedIn = (ctx: ContextInterface) => {
+  ctx.setIsLoggedIn((prev: boolean) => !prev);
+  console.log("This user is, ", ctx.isLoggedIn);
+}
+
+const onSubmitHandler = (ctxValue: ContextInterface) => {
+  ctxValue.setIsLoggedIn((prev) => !prev);
+  console.log(ctxValue.isLoggedIn);
+}
+
+const Login: React.FC = () => {
+  const ctxValue = useContext(LoginContext) as ContextInterface;
```

Comment:
could we use,

 const loginValue = useContext(LoginContext) as LoginContextInterface;